configs/sa.metrics.config.json

```diff
@@ -5,6 +5,7 @@
 	"CPUStats": false,
 	"DataCount": -1,
 	"UseSample": false,
+	"ServiceType" : "metric",
```

Comment:
Same as before. I think plural version is more natural here. Or you could just accept both.

Reply:
Fixed